cob_phidget_power_state/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cob_phidget_power_state</name>
+  <version>0.0.0</version>
```

Comment:
the release process unfortunatelly requires all packages within a repository to have the same version number :-(